index.html

```diff
@@ -16,7 +16,10 @@
        <li><a href="Gallery.html">Gallery</a></li>
        <li><a href="Services.html">Services Information</a></li>
        <li><a href="Contact.html">Contacts</a></li>
-            </ul>
+
+
+           <a href="" onclick="myFunction()"> <img id="popup" src="images/cart.jpeg">  </a>
```

Comment:
Can you clarify what the purpose of this onclick handler is, versus the onclick handler on the button whose label is "Cart" on line 81?  I'm not understanding the need for both of them.  Thanks!